README.rst

```diff
@@ -1,12 +1,51 @@
-Orange Network
-==============
+Orange3 Network
+===============
 
-Orange Network is an add-on for Orange_ data mining software package. It
-provides network visualization and analysis tools.
+Orange3-Network is an add-on for Orange_ data mining software package. It
+provides network visualization and network analysis tools.
 
 .. _Orange: http://orange.biolab.si/
 
 Documentation is found at:
 
-http://orange-network.readthedocs.org/
+http://orange3-network.rtfd.io
 
+Installation
+------------
+
+Install from Orange add-on installer through Options - Add-ons.
+
+To install the add-on with pip use
+
+    pip install orange3-network
+
+To install the add-on from source, run
+
+    python setup.py install
+
+To register this add-on with Orange, but keep the code in the development directory (do not copy it to 
+Python's site-packages directory), run
+
+    python setup.py develop
```

Comment:
Can change. However, this is in **all** current READMEs of Orange. It is in all our documentation web-wide. So I'd prefer to just add `pip install -e .` as an option with explanation to it.